apps/emqx_management/src/emqx_mgmt_api_subscriptions.erl

```diff
@@ -238,159 +238,133 @@ do_subscriptions_query_mem(QString) ->
     end.
 
 do_subscriptions_query_persistent(#{<<"page">> := Page, <<"limit">> := Limit} = QString) ->
-    Count = emqx_persistent_session_ds_router:stats(n_routes),
-    %% TODO: filtering by client ID can be implemented more efficiently:
-    FilterTopic = maps:get(<<"topic">>, QString, '_'),
-    Stream0 = emqx_persistent_session_ds_router:stream(FilterTopic),
-
+    PageSize = 100,
```

Comment:
This is a bit confusing: seems this is "internal" iterator page size, while we are in a paginated request where page size is Limit